### ISSUE TITLE
Usability for quicklisp to be viable option for cargo-make?

### ISSUE THREAD

**Issue by Kreyren** [swgillespie/rust-lisp#6]
### Context
I am doing research for https://github.com/Kreyren/kreyren/issues/47 which is looking at implementing quicklisp runner in cargo-make <https://github.com/sagiegurari/cargo-make>.

### Issue
Currently this doesn't build for me on https://github.com/swgillespie/rust-lisp/commit/704b801763db0ea54e5357fdaea98eb66c52e414 and https://github.com/swgillespie/rust-lisp/commit/0bf290ea3fe5bd7cd0668e8ca82c216f28f93d93, but based on informations in README:

![image](https://user-images.githubusercontent.com/11302521/95433216-ef7dd680-094f-11eb-869d-51af96ad0822.png)

This should have the required backend to run quicklisp <https://www.quicklisp.org/beta/>

Can you provide more info to quicklisp compatibility?

### What is quicklisp?
Basically a lisp library written to allow implementation-independent code alike:

```lisp
;; Not needed on elisp
(setf *load-verbose* nil)

(load "/home/kreyren/quicklisp/setup.lisp" :verbose nil)

(ql:quickload :uiop :silent t)

(write-line (uiop:getenv "MESSAGE"))
```

That works on clisp, ecl, sbcl and possibly elisp.

**Comment by Kreyren** [comment]
CC @swgillespie @qezz

**Comment by phoe** [comment]
Quicklisp only works on Common Lisp implementations, which rules out elisp and rust-lisp.

**Comment by Kreyren** [comment]
@phoe Thanks for info! ^-^, closing as solved then.
